backend/data/src/main/kotlin/io/tolgee/component/fileStorage/AzureFileStorageFactory.kt

```
@@ -10,19 +10,20 @@ import org.springframework.stereotype.Component
   fun create(config: AzureBlobConfig): AzureBlobFileStorage {
+    val clientBuilder = buildWithConnectionString(config.connectionString)
     try {
-      val connectionString = config.connectionString
-      val blobServiceClient =
-        BlobServiceClientBuilder()
-          .connectionString(connectionString)
-          .buildClient()
-      val containerClient = blobServiceClient.getBlobContainerClient(config.containerName)
+      val containerClient = clientBuilder.buildClient().getBlobContainerClient(config.containerName)
       return AzureBlobFileStorage(containerClient)
     } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)